### PR TITLE
[Web Bluetooth] Add Read descriptors sample

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Here's a more advanced sample for reading Bluetooth GATT descriptors.

R: @scheib 

You can try it live at:
https://beaufortfrancois.github.io/samples/web-bluetooth/read-descriptors.html
https://beaufortfrancois.github.io/samples/web-bluetooth/read-descriptors-async-await.html
https://beaufortfrancois.github.io/samples/web-bluetooth/index.html

### CODE REVIEW

File: web-bluetooth/read-descriptors-async-await.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    const descriptors = await characteristic.getDescriptors();
+
+    for (const descriptor of descriptors) {
+      switch (descriptor.uuid) {
+
+        case BluetoothUUID.getDescriptor('gatt.client_characteristic_configuration'):
+          await descriptor.readValue().then(value => {
+            log('> Client Characteristic Configuration:');
+            log('  > Notifications: ' + (value.getUint8(0) ? 'ON' : 'OFF'));
+            log('  > Indications: ' + (value.getUint8(1) ? 'ON' : 'OFF'));
```
Comment (scheib) [inline web-bluetooth/read-descriptors-async-await.js]: This is fetching the 2nd byte, but you intended to select the 2nd bit. Use something like:
 value.getUint8(0) & 0b01 // notifications
 value.getUint8(0) & 0b10 // indications

Comment (beaufortfrancois) [reply web-bluetooth/read-descriptors-async-await.js]: Rookie mistake. Thanks for catching this @scheib
ES6 binary notation to the rescue!

**Comment by beaufortfrancois** [comment]
@scheib WDYT?

**Comment by beaufortfrancois** [comment]
For info, I've just submitted a PR at https://github.com/jneen/rouge/pull/619 to make ES6 Binary literals and Octal literals highlighting works.

**Review by scheib — APPROVED** [review]

**Comment by tri-cloud** [comment]
Links giving me nothing but 404 pages someone give me a BLE Bluetooth link or download access please

**Comment by beaufortfrancois** [comment]
You can find all samples at https://googlechrome.github.io/samples/web-bluetooth/index.html

**Pull request merged**